client/src/components/MarketOrder.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+
+class MarketOrder extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      power: '$1000.00',
+      view: 'buy',
+      shares: '',
+      estim: 0,
+      reviewOrder: 'null',
```

Comment:
Maybe this was intentional, 'null' and null are different, just pointing out in case later you are expecting 'null' to be falsy

Reply:
I should probably change it to 'default' or something I just needed a third value other than 'true' or 'false' and I chose 'null' for some reason